calico-cloud_versioned_docs/version-18-2/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -77,13 +77,20 @@ You must download and set the executable flag each time you get a new version of
    ```
 ### Integrate the scanner into your build pipeline
 
-You can include the CLI scanner in your CI/CD pipelines (for example, Jenkins, Github actions). Ensure the following:
+You can include the Image Assurance CLI scanner in your CI/CD pipelines (for example, Jenkins, Github actions).
+
+1. Download the Image Assurance CLI scanner binary onto your CI runner.
+1. If you are running an ephemeral environment in the pipeline, include the download and update the executable steps in your pipeline to download the scanner on every execution.
+1. Create a secret containing the API token and API URL and make it available in the pipeline (for example, using a SECURE_API_TOKEN environment variable)
+1. Add the Image Assurance CLI scanner in your pipeline using the following command:
+
+   **Command**: `./tigera-scanner scan --apiurl <api_url> --token <secure_api_token> <image_name>:<tag>`
+
+   **Where**:
+   - <secure_api_token> is a key-value pair (variable type=secret) that is set in your pipeline
+   - <image_name> is the container image name
 
-- Download the CLI scanner binary onto your CI runner
-- If you are running an ephemeral environment in the pipeline, include the download, and update the executable steps in your pipeline to download the scanner on every execution
-- Create a secret containing the API-Token and API URL and make it available in the pipeline (for example, using a SECURE_API_TOKEN environment variable)
-- Add a step in your pipeline to run the `image-assurance-scanner` after building the container image, and specify the image name as a parameter. For example:
-        `./image-assurance-cli-scanner --apiurl <sysdig-api-url> ${IMAGE_NAME}`
+   **Example**: `./tigera-scanner scan --apiurl``xxxx://xxxx.calicocloud.io/eng` `--token $(SECURE_API_TOKEN) my-image-repo:$(tag)`
```

Comment:
We probably want the `SECURE_API_TOKEN` to match the syntax of the previous `secure_api_token`.

Reply:
Fixed.